### DISCUSSION TITLE
Inertia before event

### DISCUSSION THREAD

**Discussion by segungreat** [inertiajs/inertia#1252] [Help]
Hello .
Am trying to use inertia:before event to confirm delete action. Am using sweetalert to confirm instead of using window.confirm
The main issue is window.confirm is synchronous and before is able to wait  and get confirm true or false. 
But am unable to get it work with sweet alert. Have tried using async/await but not working.
Can anyone give me a hint or idea on how to make it work. I don't want to use the browser confirm method.

**Comment by faniadisetyawan** [comment]
up up up
